modules/apps/my-subscriptions/my-subscriptions-web/src/main/resources/META-INF/resources/js/MySubscriptionsManagementToolbarPropsTransformer.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export default function propsTransformer({portletNamespace, ...otherProps}) {
+	return {
+		...otherProps,
+		onActionButtonClick: (event, {item}) => {
+			if (item.data?.action === 'unsubscribe') {
+				const form = document.getElementById(`${portletNamespace}fm`);
+
+				if (form) {
+					form.setAttribute('method', 'post');
+
+					var subscriptionIds = form.querySelector(
```

Comment:
Maybe a `const` fit better

Reply:
Hey @boton,

That change has been made and pushed up with https://github.com/liferay-lima/liferay-portal/pull/1520/commits/5f82ad0f0fe668dd6e40e87157a696f5ddacae1e.

Please let me know if you have any questions. Thanks!